From cc4ad194a6b580591ce247ef7f8cf5905c50e100 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Wed, 3 Jan 2024 14:10:47 +0100
Subject: [PATCH] luna-init: gen-ext-timezones.py: Fix typo

This one has been there forever already, but it should really be mccInfo and not mncInfo...

Checked with legacy webOS Doctor.

Similar issue in luna-sysservice needs to be addressed.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 src/gen-ext-timezones.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gen-ext-timezones.py b/src/gen-ext-timezones.py
index 3900900..59e4e81 100755
--- a/src/gen-ext-timezones.py
+++ b/src/gen-ext-timezones.py
@@ -202,7 +202,7 @@ sysZones = list(genSysZones())
 content = {
 	'timeZone': timeZones,
 	'syszones': sysZones,
-	'mmcInfo': mccInfo
+	'mccInfo': mccInfo
 }
 
 if output is None:
